KomaMRIPlots/src/ui/DisplayFunctions.jl

```diff
@@ -1129,7 +1129,7 @@ function plot_phantom_map(
     cmin_key = get(kwargs, :cmin, factor * cmin_key)
     cmax_key = get(kwargs, :cmax, factor * cmax_key)
 
-    sort_motions!(ph.motion)
+    KomaMRIBase.initialize_motion!(ph.motion)
     t = process_times(ph.motion)
```

Comment:
Is `process_times` being exported?  I thought we changed this to `times`; what is the difference?

Reply:
No, it's a function defined inside `plot_phantom_map`  for both enlarging the number of time points in case of `intermediate_time_points > 0` and reducing them in case of `length(t) > max_time_samples `.